level1c4pps/vgac2pps_lib.py

```diff
@@ -66,7 +67,8 @@
 
 MBAND_PPS = ["M05", "M07", "M09", "M10", "M11", "M12", "M14", "M15", "M16"]
 
-MBAND_AVHRR = ["M05", "M07", "M12", "M15", "M16"]
+# "M10", "M14" are needed for NN SABAF, but are deleted before saving
```

Comment:
comments about what happens in other pars of the code should be avoided.

Reply:
I changed the comment to
 # "M10", "M14" are not AVHRR channels, but needed for NN SABAF